getting-started/setting-env.md

```diff
@@ -1,11 +1,53 @@
 ---
-title: Cosmos SDK
 order: 3
 ---
 
-# Using Cosmos SDK
+# Setting Up Environment
 
-## Connect to Demo Net
+You need an environment to run contracts. You can either run your node locally or connect to an existing network.
+If you just want to test a sample contract, We recommend you to skip running the a local node and just use a [live testnet](https://github.com/CosmWasm/tesnets)
+
+## Setup wallet
+
+You need some cash for interacting with the network. Let's set it up:
+
+```sh
+# add more wallets for testing
+wasmcli keys add fred
```

Comment:
This json format assumes you already ran some `wasmcli config` commands, which are not present here.

Either use default toml or add the `wascli config` command

Reply:
Hmmm... generally, this should be later on the "use cli" as opposed to "use repl"

The issue is that this is needed for setting up a local node.